dashing.gemspec

```diff
@@ -1,7 +1,7 @@
 # -*- encoding: utf-8 -*-
 
 Gem::Specification.new do |s|
-  s.name        = 'dashing'
+  s.name        = 'dashing-jruby'
```

Comment:
That certainly is not what I am saying... My intent in commenting was to re-spark discussion around this. There are other issues that may be resolved by moving to Puma, so I think this is good even without providing jruby compatibility.

But the gem isn't going to be renamed... People are going to type gem install dashing :+1: